pipeline/data/dataset_importer.py

```diff
@@ -90,6 +95,29 @@ def run_cmd(cmd: List[str]):
     print(result.stdout)
 
 
+def add_alignments(corpus):
+    from simalign import SentenceAligner
+
+    # We use unsupervised aligner here because statistical tools like fast_align require a large corpus to train on
+    # This is slow without a GPU and is meant to operate only on small evaluation datasets
+
+    # Use BERT with subwords and itermax as it has a higher recall and matches more words than other methods
+    # See more details in the paper: https://arxiv.org/pdf/2311.14838.pdf
```

Comment:
Did you link the correct paper here? This is all about opus trainer and the like. I don't see anything about BERT and subwords.